src/styles/Bank.module.scss

```diff
@@ -17,6 +17,11 @@
   color: $text;
   text-align: center;
   margin-bottom: 0;
+
+  &:hover {
+    color: blue;
```

Comment:
what if we use another color like #C969CD
|#C969CD|
|-|
|<img width="323" alt="Screen Shot 2020-10-23 at 8 23 03 PM" src="https://user-images.githubusercontent.com/2192588/96968617-b9ef0500-156d-11eb-9b1e-af4be56ac915.png">|